packages/assistant/src/RPA/Assistant/library.py

```diff
@@ -0,0 +1,991 @@
+import glob
+import logging
+import os
+import platform
+import subprocess
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union
+
+import flet
+from flet import (
+    Checkbox,
+    Column,
+    Container,
+    Control,
+    Dropdown,
+    ElevatedButton,
+    FilePicker,
+    FilePickerResultEvent,
+    Image,
+    Markdown,
+    Radio,
+    RadioGroup,
+    Text,
+    TextField,
+    colors,
+    icons,
+)
+from flet.control_event import ControlEvent
+from flet.dropdown import Option
+from robot.api.deco import keyword, library
+from robot.libraries.BuiltIn import BuiltIn, RobotNotRunningError
+from RPA.Assistant.flet_client import FletClient
+
+from .dialog_types import Icon, Options, Result, Size
+from .utils import optional_str, to_options
+
+
+@library(scope="GLOBAL", doc_format="REST", auto_keywords=False)
+class Assistant:
+    """This library is still in Beta and needs more testing and features.
```

Comment:
I wouldn't start the library documentation with this 😅 It can be a note later, or a highlighted banner in rpaframework.org.

Reply:
not sure how to have highlighted content on rpaframework.org and also we discussed that the Beta note should be at the beginning for users to know exactly why they shouldn't use it without our guidance. This is based on other discussions had with @kariharju and @xylix so I wouldn't change this right now